src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-details/rgw-user-details.component.html

```diff
@@ -151,5 +151,63 @@
         </tbody>
       </table>
     </div>
+    <!-- User Rate Limit -->
+      <div>
+      <legend i18n>User Rate Limit</legend>
+      <table class="cds--data-table--sort cds--data-table--no-border cds--data-table cds--data-table--md">
+        <tbody>
+          <tr>
+            <td i18n
+                class="bold w-25">Enabled</td>
+            <td class="w-75">No</td>
+            <!-- <td class="w-75">{{ user.rate_limit.enabled | booleanText }}</td> -->
+          </tr>
+          <tr>
+            <td i18n
+                class="bold">Maximum Read Ops</td>
+            <td i18n>-</td>
+            <!-- <td *ngIf="!user.rate_limit.enabled">-</td> -->
```

Comment:
Here too, you can add TODO for the commented out code!

Reply:
fixed